src/main/java/iss/DAO/ISSLocationDAO.java

```diff
@@ -7,37 +7,33 @@
 
 import java.util.List;
 
-public class ISSLocationDao implements IDAO <ISSLocation> {
+public class ISSLocationDAO {
+
     private static SessionFactory sessionFactory;
 
-    public ISSLocationDao() {
+    public void ISSLocationDao() {
         sessionFactory = DBConnector.getInstance().getSessionFactory();
     }
 
-    public Boolean create(ISSLocation issLocation) {
+    public void create(ISSLocation issLocation) {
         Session session = sessionFactory.openSession();
         Transaction transaction = session.beginTransaction();
         session.save(issLocation);
         transaction.commit();
         session.close();
-        return true;
     }
 
-    public ISSLocation loadISSLocation() {
+    public List<ISSLocation> loadISSLocation() {
         Session session = sessionFactory.openSession();
         Transaction transaction = session.beginTransaction();
-        List<ISSLocation> issLocationList = session.createQuery("SELECT * FROM craft_info", ISSLocation.class)
+        List<ISSLocation> issLocationList = session
+                .createQuery("SELECT * FROM iss_location", ISSLocation.class)
```

Comment:
In method .createQuery, query should be given in HQL, not SQL. There is a method .createSQLQuery for SQL queries.